app/ui/src/app/app.component.ts

```diff
@@ -35,7 +35,7 @@ export class AppComponent implements OnInit, AfterViewInit {
   /**
    * Logo and Icon with dark background
    */
-  logoIconDarkBg = 'assets/images/syndesis_logo_icon_darkbkg.png';
+  logoDarkBg = 'assets/images/syndesis_logo_icon_darkbkg.png';
```

Comment:
Can we not use the SVG as we are doing on the community site?

Reply:
`svg` element has been inserted, replacing `img` element.

<img width="984" alt="screen shot 2019-01-31 at 3 11 03 pm" src="https://user-images.githubusercontent.com/1402829/52082336-8767d200-256a-11e9-9235-1a6ebcb77859.png">